.gitignore

```diff
@@ -1,6 +1,2 @@
-*.pyc
-*.sqlite
 src/static/
```

Comment:
remove it also, actually `src` directory is the place, where should occurs files, that we want to track